pages/about/team.js

```diff
@@ -0,0 +1,143 @@
+        <p>
+          Lorem ipsum dolor sit ame amet, consetetur sadipscing elitr, sed diam
+          nonumy eirmod tempor, consetetur sadipscing elitr, sed diam nonumy
+          eirmod tempor invidunt voluptua.
+        </p>
+      </section>
+
+      <StyledSection>
+        <StyledArticle>
+          <StyledImage
+            src="https://avatars.githubusercontent.com/u/136329211?v=4"
+            alt="picture"
+            width={250}
+            height={250}
+          />
+
+          <h3>Lüder Budde </h3>
+          <StyledSocialsContainer>
+            <Link
+              href="https://www.linkedin.com/in/luederbudde/"
+              target="_blank"
+            >
+              <StyledIcon variant="linkedIn" />
+            </Link>
+            <Link href="https://github.com/luederb" target="_blank">
+              <StyledIcon variant="gitHub" />
+            </Link>
+          </StyledSocialsContainer>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy em ipsum dolor sit amet, consetetur sadipscing elitr, sed
+            diam nonumy eirmod tempor em ipsum dolor sit amet, consetetur
+            sadipscing elitr, sed diam eirmod tempor invidunt ut labore , sed
+            diam voluptua.invidunt ed diam voluptuaem ipsum dolor sit amet.
+          </p>
+        </StyledArticle>
+
+        <StyledArticle>
+          <StyledImage
+            src="https://avatars.githubusercontent.com/u/120521307?v=4"
+            alt="picture"
+            width={250}
+            height={250}
+          />
+
+          <h3>Lennart Kaminsky</h3>
+          <StyledSocialsContainer>
+            <Link
+              href="https://www.linkedin.com/in/lennart-kaminsky/"
+              target="_blank"
+            >
+              <StyledIcon variant="linkedIn" />
+            </Link>
+            <Link href="https://github.com/lennart-kaminsky" target="_blank">
+              <StyledIcon variant="gitHub" />
+            </Link>
+          </StyledSocialsContainer>
+          <p>
+            Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam
+            nonumy em ipsum dolor sit amet, consetetur sadipscing elitr, sed
+            diam nonumy eirmod tempor em ipsum dolor sit amet, consetetur
+            sadipscing elitr, sed diam eirmod tempor invidunt ut labore , sed
+            diam voluptua.invidunt ed diam voluptuaem ipsum dolor sit amet.
+          </p>
+        </StyledArticle>
+
+        <StyledArticle>
+          <StyledImage
+            src="https://avatars.githubusercontent.com/u/134974986?s=400&u=3ceb33fd1cf234d959ac32a417aeacdcbc1bbcfd&v=4"
+            alt="picture"
+            width={250}
+            height={250}
+          />
+
+          <h3>Gökhan Agkurt</h3>
+          <StyledSocialsContainer>
+            <Link
+              href="https://www.linkedin.com/in/g%C3%B6khan-agkurt-21b58b230/"
+              target="_blank"
+            >
```

Comment:
The next/link component in Next.js is primarily designed for internal navigation within your Next.js application. It provides optimized client-side navigation between pages, fetching and rendering only the necessary components. For external links, you should use the standard HTML <a> element.

Reply:
Note that for external links, it's a good practice to include the target="_blank" attribute to open the link in a new tab and add rel="noopener noreferrer" for security reasons (preventing potential security vulnerabilities related to opening new tabs).

Reply:
Thanks a lot for this clarification! We actually didn't really know about this difference 🙈. We also added the target and rel properties to the a-tags 👍